app/src/main/java/com/xinto/opencord/domain/emoji/DomainEmoji.kt

```diff
@@ -0,0 +1,29 @@
+package com.xinto.opencord.domain.emoji
+
+import androidx.compose.runtime.Immutable
+import com.github.materiiapps.partial.getOrNull
+import com.xinto.opencord.rest.models.emoji.ApiEmoji
+
+/**
+ * A unique identifier of a [DomainEmoji] that is obtained through [DomainEmoji.identifier]
+ */
+typealias DomainEmojiIdentifier = Int?
```

Comment:
This could instead be a value class if we want to reject  implicit integer to identifier assignments

Reply:
wdym elaborate

Reply:
resolved in discord